sdk/formrecognizer/Azure.AI.FormRecognizer/src/FieldValue.cs

```diff
@@ -442,5 +471,154 @@ public FieldValueGender AsGender()
 
             return _fieldValue.ValueGender.Value;
         }
+
+        /// <summary>
+        /// Converts the <see cref="FieldValue"/> instance to the specified type
+        /// specified in the parameter result.
+        /// </summary>
+        /// <returns>A bool value indicates whether the operation succeeded.</returns>
+        public bool TryParse<T>(out T result)
+        {
+            bool parse = true;
+            switch (ValueType)
+            {
+                case FieldValueType.String:
+                    try
+                    {
+                        result = (T)(object)AsString();
+                        if (result == null)
+                        {
+                            result = default;
+                            parse = false;
+                        }
```

Comment:
In the case of strings, `AsString` will return `null` if there is no value in `ValueString`. I wanted to change it to throw an exception, but this will be a breaking change from GA version... and because it is a string, not sure it is worth it.
But when talking about `TryParse` if the value is `null`, then in my head it means it didn't work, so I prefer to return `false` and `null`.
- Triple checking, fixing `AsString` to throw an exception if `ValueString` is not provided, while before we didn't and just returned null, is a breaking change, right?
- Do you think this is inconsistent? 
- Should I follow the behavior of as `AsString` and let `TryParse` return `true` and `null`?